docs/hub/models-widgets.md

```diff
@@ -2,23 +2,26 @@
 
 ## What's a widget?
 
-Many model repos have a widget that allows anyone to run inferences directly in the browser!
+Many model repos have a widget that allows anyone to run inferences directly in the browser. These widgets are powered by [Inference Providers](https://huggingface.co/docs/inference-providers), which provide access to hundreds of machine learning models through multiple AI infrastructure providers.
```

Comment:
```suggestion
Many model repos have a widget that allows anyone to run inferences directly in the browser. These widgets are powered by [Inference Providers](https://huggingface.co/docs/inference-providers), which provide developers streamlined, unified access to hundreds of machine learning models, backed by our serverless inference partners.
```

(closer terminology to the one-liner we use in https://huggingface.co/docs/inference-providers/index)